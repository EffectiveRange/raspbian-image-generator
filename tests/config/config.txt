# For more options and information see
# http://rpf.io/configtxt

[all]
