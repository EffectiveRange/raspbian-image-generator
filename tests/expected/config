IMG_NAME=test-target
TARGET_HOSTNAME=test-target
DEPLOY_COMPRESSION=xz
ENABLE_SSH=1
CLEAN=1
STAGE_LIST="stage0 stage1 stage2"
