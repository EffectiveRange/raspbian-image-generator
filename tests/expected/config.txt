# For more options and information see
# http://rpf.io/configtxt

[all]
option3
option4
